src/main/scala/com/target/data_validator/CliOptionParser.scala

```diff
@@ -0,0 +1,50 @@
+package com.target.data_validator
+
+import scopt.OptionParser
+
+case class CmdLineOptions(
```

Comment:
I assume you want to be backward compatible to reduce the number of changes in this PR, but this would be better named "CliOptions" to be consistent with "CliOptionParser"

Reply:
I think it's pretty safe to effect that sensible change in this PR. Done.